addons/project/models/project.py

```diff
@@ -26,6 +26,7 @@
     'project_id',
     'display_project_id',
     'color',
+    'should_display_partner',
```

Comment:
In project sharing feature, no private tasks are displayed, so I don't think this field is useful in project sharing. 🙂

Reply:
I added `should_display_partner` to avoid so many view inheritance just to change partner attrs :slightly_smiling_face: